cloud-interfaces/api/cloudimages-api.rst

```diff
@@ -142,10 +141,10 @@ In the sample response,
 correspond to the information available on the Cloud Control Panel.
 
 In the
-:rax-docs:`Getting Started Guide for the Cloud Images API <images/api/v2/ci-gettingstarted>`,
-you can see an example of
-:rax-docs:`obtaining a list of images by using the cURL command-line interface (CLI)
-<images/api/v2/ci-gettingstarted/content/list-images.html>`.
+:rax-docs:`Getting Started Guide for the Cloud Images <cloud-images/v2/getting-started/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing images
+<cloud-images/v2/getting-started/use-images/#listing-images>`.
 
 Learn about Cloud Images in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
those were definitely broken, though i swear i tested them.  should be fixed now